src/domain/Popovers/DropdownMenu/subcomponents/Section.tsx

```diff
@@ -116,6 +146,42 @@ class Section extends React.PureComponent<ISectionProps, never> {
     )
   }
 
+  public static defaultProps: Partial<ISectionProps> = {
+    sectionType: 'default',
+    closeDropdownBehaviour: 'whenActionProvided'
+  }
+
+  public render (): JSX.Element {
+    const {
+      tooltipMessage,
+      tooltipProps
+    } = this.props
+
+    if (tooltipMessage) {
+      return (
+        <TooltipPopover
+          toggleComponent={this.toggleComponent(this.content)}
+          {...tooltipProps}
```

Comment:
Can we add default anchor positions and width here pls?
``
    parentAnchorPosition={{
      xPos: 'left',
      yPos: 'top'
    }}
    popoverAnchorPosition={{
      xPos: 'right',
      yPos: 'top'
    }}
   width={300}
``